qa-tests-backend/src/test/groovy/IntegrationsTest.groovy

```diff
@@ -749,35 +755,30 @@ class IntegrationsTest extends BaseSpecification {
         }       | StatusRuntimeException | /PermissionDenied/ | "incorrect project"
     }
 
-    // TODO disabled due to flaking (ROX-7902), shoudl be re-enabled once reworked (ROX-10310)
-    //@Tag("Integration")
-    //def "Verify syslog notifier"() {
-    //    given:
-    //    "the some syslog receiver is created"
-    //    // Change the local port numbers so we don't conflict with any other splunk instances
-    //    SplunkUtil.SplunkDeployment splunkDeployment = SplunkUtil.createSplunk(orchestrator,
-    //            Constants.ORCHESTRATOR_NAMESPACE, true)
-
-    //    when:
-    //    "call the grpc API for the syslog integration."
-    //    SyslogNotifier notifier = new SyslogNotifier(splunkDeployment.syslogSvc.name, 514,
-    //            splunkDeployment.splunkPortForward.localPort)
-    //    try {
-    //        notifier.createNotifier()
-    //    } catch (Exception e) {
-    //        throw new AssumptionViolatedException("Could not create syslog notifier. Skipping test!", e)
-    //    }
-
-    //    then:
-    //    "Verify the messages are seen in the json"
-    //    // We should have at least one audit log for the message which created the syslog integration.
-    //    notifier.validateViolationNotification(null, null, false)
-
-    //    cleanup:
-    //    "remove splunk and syslog notifier integration"
-    //    SplunkUtil.tearDownSplunk(orchestrator, splunkDeployment)
-    //    notifier.deleteNotifier()
-    //}
+    @Unroll
+    @Tag("Integration")
+    def "Verify syslog notifier"() {
+       given:
+       "the some syslog receiver is created"
+       def syslog = Rsyslog.createRsyslog(orchestrator, Constants.ORCHESTRATOR_NAMESPACE)
+
+       and:
+       "call the grpc API for the syslog integration."
+       SyslogNotifier notifier = new SyslogNotifier(syslog.syslogSvc.name, 514)
+       notifier.createNotifier()
+
+        when:
+       "Verify syslog connecion is succesful"
+       assert notifier.testNotifier()
+
+        then:
+        "add verification that syslog received alert"
+
+        cleanup:
+        "remove syslog notifier integration"
```

Comment:
and tear down rsyslog server

Reply:
I have added syslog.tearDown(orchestrator) in cleanup